test/select.spec.js

```diff
@@ -2504,6 +2504,14 @@ describe('ui-select tests', function() {
       expect($(el).scope().$select.selected.length).toBe(3);
     });
 
+    it('should split pastes on ENTER, even though ENTER is not first taggingToken, but not try another token', function() {
```

Comment:
Can you turn this into two tests, one for each bit of functionality?
Example:

1. `it('should split pastes on token that isn't the first token',...)`
2. `it('should split pastes only on first tagging token found in paste string', ...)`